collections/Lyell/MS_Lyell_50.xml

```diff
@@ -122,56 +237,112 @@
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 63v)</locus>
+                        <note>After the text of the <title>Elucidarius</title>, the scribe has written in red capitals: <quote>
+                           <lb/>Auscultate pie prescriptę uerba sophyę.
+                           <lb/>Auribus audite cordeque diligite.</quote></note>
+                     </msItem>
+                     <msItem>
+                        <note>Various later additions on fol. 63v.</note>
+                        <msItem>
+                           <quote>
+                              <lb/>Diues mendicus deus infernus paradysus
+                              <lb/>Pascit deposcit conpensat torquet amenat</quote>
+                           <bibl>Walther, <title>Initia</title>, no. 4624</bibl>
+                        </msItem>
+                        <msItem>
+                           <note>Τwo lines in mirror writing</note>
+                           <quote>
+                              <lb/>Te diligo pre ceteris, a nexu pie Veneris me soluere dignare. 
+                              <lb/>Nam solam te desidero, spem puram ad te defero quam nollem euitare</quote>
+                        </msItem>
+                        <msItem>
+                           <note>A charm</note>
+                           <quote>✠ Esyppus ✠ Speusippus ✠ Melesyppus ✠ Caspar ✠ Melchior ✠ Balthasar; d: Incaustum dum penna probat simul ipsa probatur</quote>
+                           <note>See W. Wattenbach, <title>Das Schriftwesen im Mittelalter</title>, 3rd ed., Leipzig, 1896, p. 231.</note>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <note>On the verso of a strip of parchment used in the binding, on which is written another charm (13th cent.), mostly illegible (beg. : <quote>✠ In nomine patris</quote>, ends: <quote>protectio huic N. famulo dei ✠</quote>) is written <quote>Casperus Mulhoff</quote> (15th cent.?); on the front and end pastedowns are also written the letters <quote>C.M.</quote> and on the end pastedown are <quote>1488. K [or R] Petri</quote> and the line: <quote>Ante dei uultum n[il] unquam transit inultum</quote>. There are annotations in the hand of this last note, which may be a hand used in one of the other notes, throughout the <title>Elucidarius</title>; and on fol. 4–4v of the first MS. this hand has supplied words cropped by the binder.</note>
```

Comment:
I think Casper Mulhoff should be marked up as a `persName` and given an authority record. From http://18.235.151.129/detail.php?msid=9508 he appears to have been prior of Admont.

Reply:
I have not found any other useful sources for his dates, if you would not mind creating the record.

Reply:
done: person_3379